src/pudl/metadata/resources/gridpathratoolkit.py

```diff
@@ -0,0 +1,52 @@
+"""Table definitions for data coming from the GridPath Resource Adequacy Toolkit."""
+
+from typing import Any
+
+RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "core_gridpathratoolkit__hourly_aggregated_extended_capacity_factors": {
```

Comment:
zane and I chatted about this name and came to `out_gridpathratoolkit__hourly_potential_capacity_factors` (potential or available either one! or some other more standard name for generation profiles).

because this really is a processed output table (we just happen to be building it in a lightweight ETL) but it is processed by GP.